templates/prometheus/k8s-api-endpoints.yaml.j2

```diff
@@ -0,0 +1,20 @@
+job_name: 'k8s-api-endpoints'
+kubernetes_sd_configs:
+- api_server: https://{{k8s_api_endpoint}}
+  role: endpoints
+  tls_config:
+    insecure_skip_verify: true
```

Comment:
:open_mouth:

Reply:
There is a separate bug about improving auth and security for this scraper, and I laid the groundwork for addressing both of those in the interface layers, but forgot to follow-up on it in this charm.  I'll take a stab at it.